openshift/pipeline/config-prod.groovy

```diff
@@ -332,7 +332,7 @@ environments {
     'prod' {
         vars {
             DB_PVC_SIZE = '50Gi'
-            DOCUMENT_PVC_SIZE = '390Gi'
+            DOCUMENT_PVC_SIZE = '80Gi'
```

Comment:
Is this based on a legitimate value of projected use until we switch over to s3?

Reply:
No, but one nice thing is that the new netapp pvcs can scale to larger sizes if we need to. This size was created because of the quota limit with the two PVCs existing in parallel. 

At present we're only using 15gb of document storage.